app/controllers/users.js

```diff
@@ -1,15 +1,19 @@
 const userService = require('../services/users');
+const { databaseError } = require('../errors');
+const errorMsg = require('../constants/errorMessages');
+const varParse = require('../serializers/varParse');
 
 exports.signUp = async (req, res) => {
   try {
-    const userCreated = await userService.createUser({
+    const userCreated = await varParse.nameParse({
       firstName: req.body.first_name,
       lastName: req.body.last_name,
       email: req.body.email,
       password: req.body.password
     });
```

Comment:
- This function is not an asynchronous operation (remove async-await).
- Just send to the function `req.body`.